raiden/network/rpc/client.py

```diff
@@ -509,18 +509,18 @@ def __init__(
 
         result = jsonrpc_client.call(
             'eth_getCode',
-            address_encoder(token_address),
+            token_address,
             'latest',
         )
 
         if result == '0x':
             raise ValueError('Token address {} does not contain code'.format(
-                address_encoder(token_address),
+                token_address,
             ))
 
-        proxy = jsonrpc_client.new_abi_contract(
+        proxy = jsonrpc_client.new_contract_proxy(
```

Comment:
nice change. Did not know that `new_abi_contract` is deprecated in favour of `new_contract_proxy`.
But it seems to be so acccording to: https://github.com/ethereum/pyethapp/blob/afc8f66b8e47d6b21ea780e754c300a52b32411d/pyethapp/rpc_client.py#L189